### PR TITLE
[MONIT-40972 ] lz4-pure-java

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: proxy/pom.xml [hunk 1]
```diff
@@ -381,6 +381,11 @@
         <artifactId>okhttp</artifactId>
         <version>4.9.3</version>
       </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.24.0</version>
+        </dependency>
```
Comment (locke-chappel) [inline proxy/pom.xml:388]: Isn't commons-compress already included on line 334 above?

**Comment by locke-chappel** [comment]
Overall, looks good from the security side - old lz4 library is removed and implementation now uses commons-compress.

File: proxy/pom.xml (hunk 1 above)
Comment (laullon) [reply proxy/pom.xml:388]: yeah... quick test ;-)

Comment (locke-chappel) [reply proxy/pom.xml]: looks good now :)

**Pull request merged**